docs/_articles/en/v3.x/index.md

```diff
@@ -36,30 +36,21 @@ lang: en
 <br>
 -->
 
-## Salesforce Code Analyzer Plug-in
+## Salesforce Code Analyzer
```

Comment:
The banners in this file will need to be changed to indicate that 2.x is end-of-lifed and 3.x is the main release.